deploy/operator/utils.sh

```diff
@@ -51,6 +51,16 @@ function disk_to_wwn() {
     echo "0x0$(hash ${1} 15)"
 }
 
+function wait_for_object() {
```

Comment:
Doesn't ``wait_for_object_amount`` do the same (and with timeout mechanism)?

Reply:
Oh, havent seen this function. You're right .... Changed!